features/application.feature

```diff
@@ -1,4 +1,4 @@
-Feature: User visits the home page
+Feature: I visit the site the home page
```

Comment:
change `I visit the site the home page` to `I visit the site`

Reply:
@magnus-thor fixed! 😄